cfg/cfg.go

```diff
@@ -82,7 +85,30 @@ func init() {
 	}
 	flag.StringVar(&AuthenticatedUserID, "authenticated-user-id", defaultAuthenticatedUserID, "OpenShift user's ID that should has access to API. Is used only if useBearerToken is configured")
 
-	flag.DurationVar(&IdleTimeout, "idle-timeout", -1*time.Nanosecond, "IdleTimeout is a inactivity period after which workspace should be stopped. Examples: -1, 30s, 15m, 1h")
+	defaultIdleTimeout := -1 * time.Nanosecond
```

Comment:
Should it be `900 * time.Second` ?
Could we move default to const to have something like this `idleTimeout := defaultIdleTimeout`

Reply:
> Should it be 900 * time.Second

I don't have strong opinions on this default value since the value should come from the environment variable anyways, but I updated it to  `1800 * time.Second` to be consistent with the default value on the che-operator side here: https://github.com/eclipse-che/che-operator/pull/1443

> Could we move default to const to have something like this idleTimeout := defaultIdleTimeout

Sounds good, I made an update to this PR

 WDYT?